src/app/docs/kagent/crds/crds.mdx

```diff
@@ -0,0 +1,32 @@
+# Kagent Custom Resource Definitions (CRDs)
+
+## What are CRDs?
+
+Custom Resource Definitions (CRDs) in Kubernetes provide a mechanism to extend the Kubernetes API by defining new, custom resource types. These custom resources behave similarly to built-in Kubernetes objects like **Pods** or **Deployments**, but are tailored to specific application or organizational needs.
+
+---
+
+### Installing Kagent CRDs
```

Comment:
this installing section might be redundant here -- instead, in the List of CRDs, you could say something like: "Once you've followed the quickstart and installed kagent, oyu can use `kubectl get crd ...` to list all CRDs"